bridge/articles/bridge-to-kubernetes-aks.md

```diff
@@ -43,174 +43,109 @@ az aks create \
 
-![Logging output](media/bridge-to-kubernetes-vs-code/output.png)
+Make a request to the stats-api by choosing the **stats** link.
 
-Click on the **Kubernetes** Status bar and choose **Show connection diagnostics information**. This command prints the current environment variables and DNS entries in the logging output.
+![Running web site - choose the status link](media/bridge-to-kubernetes-sample/stats.png)
 
-![Output with diagnostics](media/bridge-to-kubernetes-vs-code/output-diagnostics.png)
+Notice the traffic that initially started in your cluster was redirected to your locally running version (outside of the cluster) where the breakpoint was triggered.
 
-Additionally, you can find the diagnostic logs in the `Bridge to Kubernetes` directory in your development computer's TEMP directory. On Windows 10, that's in `%TEMP%\Bridge to Kubernetes`. On a Mac, the TEMP directory can be found by running `echo $TMPDIR` from a terminal window. On Linux, it is `/tmp/Bridge to Kubernetes`.
+Press play and let the request continue complete transparently.
 
-## Running in isolation mode
+## Clean up
 
-With Bridge to Kubernetes, you can also set up an isolated version the services you're working on, meaning that others who are using the cluster won't be affected by your changes. This isolation mode is accomplished by routing your requests to your copy of each affected service, but routing all other traffic normally. More explanation on how this is done can be found at [How Bridge to Kubernetes Works][btk-overview-routing].
+When you stop the debugger all changes to the cluster are reverted. Note, if you were running with isolation on, the routing manager pod will be left there to expedite the start of the debugger the next time around.
 
-## Remove the sample application from your cluster
-
-Use the provided script to remove the sample application from your cluster.
+You can leave your AKS cluster on for future debugging or you can delete it with below command.
 
 ```azurecli-interactive
-./bridge-quickstart.sh -c -g MyResourceGroup -n MyAKS
+az aks delete --name MyAKS --resource-group MyResourceGroup
 ```
-
-## Troubleshooting
-
- If you get this error when activating the Bridge to Kubernetes extension:
-
-"Failed to update dependencies: maximum number of retries exceeded"
-
- First, retry the activation using the button. If it repeatedly does not succeed, see [https://github.com/microsoft/mindaro/issues/32](https://github.com/microsoft/mindaro/issues/32).
-
-## Next steps
-
-Learn more about Bridge to Kubernetes at [How Bridge to Kubernetes works][btk-how-it-works].
-
-[azure-kubernetes-service]: /azure/aks/kubernetes-walkthrough
-[azds-cli]: /azure/dev-spaces/how-to/install-dev-spaces#install-the-client-side-tools
-[azds-tmp-dir]: /azure/dev-spaces/troubleshooting#before-you-begin
-[btk-vs-code]: https://marketplace.visualstudio.com/items?itemName=mindaro.mindaro
-[azure-cli]: /cli/azure/install-azure-cli?view=azure-cli-latest&preserve-view=true
```

Comment:
Some of these links are used in the document, so you can't delete them from this table. For example, [azure-cli] is used in line 24